src/components/SingleDatePicker.jsx

```diff
@@ -218,6 +218,12 @@ export default class SingleDatePicker extends React.Component {
       values(visibleDays).forEach((days) => {
         Object.keys(days).forEach((day) => {
           const momentObj = moment(day);
+          if (isOutsideRange(momentObj)) {
```

Comment:
i have to admit my memory is a bit fuzzy on this now but I believe isBlocked adds its own modifier on top of the others. So you can successfully remove the modifiers for isDayBlocked and isOutsideRange but still have the calendar day blocked. This is what i remember from my own pr, hopefully one of the maintainers can clarify.